code/style.css

```diff
@@ -1,4 +1,379 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/* MOBILE VIEW <-- 768PX */
+
+body {
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+    font-family: 'Noto Serif JP', serif;
+    text-align: justify;
 }
+
+h1 {
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 400;
+    color: #214273;
+}
+
+h2 {
+    font-family: 'Noto Serif JP', serif;
+    color: #214273;
+    margin: auto;
+    line-height: 1.5;
+}
+
+h5 {
+    font-family: 'Montserrat', sans-serif;
+    font-size: 15px;
+    color: #214273;
+    font-weight: 400;
+}
+
+p {
+    font-family: 'Montserrat', sans-serif;
+    color: #214273;  
+    line-height: 1.5;  
+}
+
+/* HEADER */
+
+.header-parent {
+    width: 100%;
+    height: 100px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding-top: 10px;
+    box-sizing: border-box;
+}
+
+.logo-title{
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-left: 10px;
+}
+
+.logo-title h1 {
+    margin-left: 10px;
+    font-size: 28px;
+}
+
+.hamburger-menu {
+    width: 30px;
+    height: 30px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin: 24px;
+    box-sizing: border-box;
+}
+    
+.hamburger-menu div {
+    width: 30px;
+    height: 3px;
+    background: #214273;
+    border-radius: 2px;
+}
+
+.header-parent nav {
+    display: none;
+}
+
+
+/* MAIN ARTICLE */
+
+.main-article-parent {
+    width: 100%;
+    background:white;
+    padding-bottom: 10px;
+}
+
+.main-article-child {  
+    width: 100%;
+    padding: 50px 15px;
+    background: rgb(255, 215, 222); 
+    box-sizing: border-box;  
+    margin: auto; 
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
```

Comment:
Since there's only one main article perhaps you could combine the styling for these two classes into one, something like this:
```
.main-article-parent {
    width: 100%;
    padding: 50px 15px 10px 15px;
    background: rgb(255, 215, 222); 
    box-sizing: border-box;  
    margin: auto; 
    display: flex;
    flex-direction: column;
    justify-content: center;
}
```